src/openrct2/ride/Vehicle.cpp

```diff
@@ -14,575 +14,492 @@
-    { 0, 0 },   // SOUND_LAUGH_1
-    { 0, 0 },   // SOUND_LAUGH_2
-    { 0, 0 },   // SOUND_LAUGH_3
-    { 0, 0 },   // SOUND_APPLAUSE
-    { 0, 0 },   // SOUND_HAUNTED_HOUSE_SCARE
-    { 0, 0 },   // SOUND_HAUNTED_HOUSE_SCREAM_1
-    { 0, 0 },   // SOUND_HAUNTED_HOUSE_SCREAM_2
-    { 0, 0 },   // SOUND_48
-    { 0, 0 },   // SOUND_49
-    { 0, 0 },   // SOUND_ERROR
-    { 0, 0 },   // SOUND_51
-    { 1, 0 },   // SOUND_LIFT_4
-    { 1, 0 },   // SOUND_LIFT_5
-    { 1, 0 },   // SOUND_TRACK_FRICTION_2
-    { 1, 0 },   // SOUND_LIFT_6
-    { 1, 0 },   // SOUND_LIFT_7
-    { 1, 2 },   // SOUND_TRACK_FRICTION_3
-    { 0, 1 },   // SOUND_SCREAM_8
-    { 0, 1 },   // SOUND_TRAM
-    { 0, 0 },   // SOUND_DOOR_OPEN
-    { 0, 0 },   // SOUND_DOOR_CLOSE
-    { 0, 0 }    // SOUND_62
+    { 1, 0 }, // SOUND_LIFT_1
+    { 1, 0 }, // SOUND_TRACK_FRICTION_1
+    { 1, 0 }, // SOUND_LIFT_2
+    { 0, 1 }, // SOUND_SCREAM_1
+    { 0, 0 }, // SOUND_CLICK_1
+    { 0, 0 }, // SOUND_CLICK_2
+    { 0, 0 }, // SOUND_PLACE_ITEM
+    { 0, 1 }, // SOUND_SCREAM_2
+    { 0, 1 }, // SOUND_SCREAM_3
+    { 0, 1 }, // SOUND_SCREAM_4
+    { 0, 1 }, // SOUND_SCREAM_5
+    { 0, 1 }, // SOUND_SCREAM_6
+    { 1, 0 }, // SOUND_LIFT_3
+    { 0, 0 }, // SOUND_PURCHASE
+    { 0, 0 }, // SOUND_CRASH
+    { 0, 0 }, // SOUND_LAYING_OUT_WATER
+    { 0, 0 }, // SOUND_WATER_1
+    { 0, 0 }, // SOUND_WATER_2
+    { 0, 1 }, // SOUND_TRAIN_WHISTLE
+    { 0, 1 }, // SOUND_TRAIN_CHUGGING
+    { 0, 0 }, // SOUND_WATER_SPLASH
+    { 1, 0 }, // SOUND_HAMMERING
+    { 0, 0 }, // SOUND_RIDE_LAUNCH_1
+    { 0, 0 }, // SOUND_RIDE_LAUNCH_2
+    { 0, 0 }, // SOUND_COUGH_1
+    { 0, 0 }, // SOUND_COUGH_2
+    { 0, 0 }, // SOUND_COUGH_3
+    { 0, 0 }, // SOUND_COUGH_4
+    { 1, 0 }, // SOUND_RAIN_1
+    { 0, 0 }, // SOUND_THUNDER_1
+    { 0, 0 }, // SOUND_THUNDER_2
+    { 1, 0 }, // SOUND_RAIN_2
+    { 1, 0 }, // SOUND_RAIN_3
+    { 0, 0 }, // SOUND_BALLOON_POP
+    { 0, 0 }, // SOUND_MECHANIC_FIX
+    { 0, 1 }, // SOUND_SCREAM_7
+    { 0, 0 }, // SOUND_TOILET_FLUSH
+    { 0, 0 }, // SOUND_CLICK_3
+    { 0, 0 }, // SOUND_QUACK
+    { 0, 0 }, // SOUND_NEWS_ITEM
+    { 0, 0 }, // SOUND_WINDOW_OPEN
+    { 0, 0 }, // SOUND_LAUGH_1
+    { 0, 0 }, // SOUND_LAUGH_2
+    { 0, 0 }, // SOUND_LAUGH_3
+    { 0, 0 }, // SOUND_APPLAUSE
+    { 0, 0 }, // SOUND_HAUNTED_HOUSE_SCARE
+    { 0, 0 }, // SOUND_HAUNTED_HOUSE_SCREAM_1
+    { 0, 0 }, // SOUND_HAUNTED_HOUSE_SCREAM_2
+    { 0, 0 }, // SOUND_48
+    { 0, 0 }, // SOUND_49
+    { 0, 0 }, // SOUND_ERROR
+    { 0, 0 }, // SOUND_51
+    { 1, 0 }, // SOUND_LIFT_4
+    { 1, 0 }, // SOUND_LIFT_5
+    { 1, 0 }, // SOUND_TRACK_FRICTION_2
+    { 1, 0 }, // SOUND_LIFT_6
+    { 1, 0 }, // SOUND_LIFT_7
+    { 1, 2 }, // SOUND_TRACK_FRICTION_3
+    { 0, 1 }, // SOUND_SCREAM_8
+    { 0, 1 }, // SOUND_TRAM
+    { 0, 0 }, // SOUND_DOOR_OPEN
+    { 0, 0 }, // SOUND_DOOR_CLOSE
+    { 0, 0 }  // SOUND_62
 };
 
 static const uint8 SpaceRingsTimeToSpriteMap[] = {
```

Comment:
Probably better to split off all these maps to a separate _data.c(pp) file some time. But that's not something for this PR, though.